src/index.html

```diff
@@ -1,14 +1,37 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="menu">
+
+      <a class="logo" href="#home">
+        <img src="moyo-logo.png" alt="moyo-header"
+        height="40px" width="40px">
+      </a>
+
+      <nav class="nav">
+        <ul>
+          <li><a class="nav__link is-active "href="#apple">apple</a></li>
+          <li><a class="nav__link is-active" href="#samsung">samsung</a></li>
+          <li><a class="nav__link is-active" href="#smpartphones">smpartphones</a></li>
+          <li><a class="nav__link is-active" href="#laptops & computers">laptops & computers</a></li>
+          <li><a class="nav__link is-active" href="#gadgets">gadgets</a></li>
+          <li><a class="nav__link is-active" href="#tablets">tablets</a></li>
+          <li><a class="nav__link is-active" href="#photo">photo</a></li>
+          <li><a class="nav__link is-active" href="#video">video</a></li>
```

Comment:
1.due to checklist only first  link should have class is-active
2. please make links from new line: 
```suggestion
          <li>
            <a class="nav__link is-active "href="#apple">apple</a>
          </li>
     
```